readme.md

```diff
@@ -107,11 +118,9 @@ There are no requirements to simply add a post/fix and push to GitHub for a rend
 
 #### Installing Hugo for local preview
 
-The current version of `hugo` on the server uses is defined in [netlify.toml](netlify.toml). To install hugo locally:
+The current version of Hugo on the server uses is defined in [netlify.toml](netlify.toml). 
```

Comment:
Should this say
The version of Hugo used by the server is defined in [netlify.toml](netlify.toml). 

Text error in the original makes meaning unclear.